src/app/window-handler.ts

```diff
@@ -440,7 +440,7 @@ export class WindowHandler {
     /**
      * Move window to the same screen as main window
      */
-    public moveWindow(windowToMove: BrowserWindow) {
+    public moveWindow(windowToMove: BrowserWindow, fixedYPosition?: number) {
         if (this.mainWindow && windowExists(this.mainWindow)) {
             const display = electron.screen.getDisplayMatching(this.mainWindow.getBounds());
```

Comment:
@johankwarnmarksymphony I think for screen sharing indicator we should go with focused window rather than `mainWindow` as the RTC can be a popped-out and be placed in a different screen compared with the main window right?

Reply:
I think the designer wants it always on the screen where the main symphony app is

Reply:
@KiranNiranjan Interesting idea, to follow the pop out/focused window.
But let us go with Symphony-app for now as the designer wants